app/models/user.rb

```diff
@@ -20,6 +20,11 @@ class User < ActiveRecord::Base
   has_many :outgoing_delegations, class_name: 'ApprovalDelegate', foreign_key: 'assigner_id'
   has_many :outgoing_delegates, through: :outgoing_delegations, source: :assignee
 
+  has_many :incoming_delegations, class_name: 'ApprovalDelegate', foreign_key: 'assignee_id'
+  has_many :incoming_delegates, through: :incoming_delegations, source: :assigner
```

Comment:
why are we defining these now? do not appear to be used anywhere else in this PR, as far as I can tell